templates/zerver/integrations/irc.md

```diff
@@ -16,9 +16,14 @@ Mirror an IRC channel in Zulip!
 1. Activate the virtualenv by running the `source` command printed
    at the end of the output of the previous step.
 
-1. Install the bridge software in your virtualenv, by running:
+1. Go to the directory containing the bridge script if you aren't already so
```

Comment:
This also seems a bit incorrect. Probably something like `Go to the directory containing the bridge script if you haven't already done so`.

Reply:
Fixed.